templates/patterns/date_block/date_block.ui_patterns.yml

```diff
@@ -0,0 +1,24 @@
+date_block:
+  label: "Date Block"
+  description: "Date block allows to present a date block that can be placed in other components."
+  variants:
```

Comment:
Not needed at this moment. A single variant if not a variant is default behaviour.